lib/src/analyzers/unused_files_analyzer/reporters/reporters_list/console/lint_console_reporter.dart

```diff
@@ -0,0 +1,20 @@
+import 'dart:io';
```

Comment:
@incendial a bit strange, the file named `lint_console_reporter.dart`, with contained `UnusedFilesConsoleReporter` class

Reply:
Fixed.